### ISSUE TITLE
document two typical user journeys & link them to current & needed functionality

### ISSUE THREAD

**Issue by mdingemanse** [elpaco-escience/talkr#68]
Add these in the form of sections in the workflow.Rmd vignette.

**Comment by bvreede** [comment]
Code used in developing `add_lines`:

```r

conv_token <- conv |>
  tokenize() |>
  add_lines(time_columns="relative_time")


conv |>
  add_lines() |>
  ggplot(aes(x = line_end, y = line_participant)) +
  geom_turn(aes(
    begin = line_begin,
    end = line_end)) +
  geom_token(data = conv_token,
             aes(x = line_relative_time,
                 y = line_participant,
                 color = rank)) +
  xlab("Time (ms)") +
  ylab("") +
  theme_turnPlot()

```

**Comment by mdingemanse** [comment]
some additions

```conv <- ifadv::ifadv[ifadv::ifadv$source == "/dutch2/DVA10O",]

conv_token <- conv |>
  tokenize() |>
  add_lines(time_columns="relative_time")

conv |>
  add_lines() |>
  ggplot(aes(x = line_end, 
             y = line_participant)) +
  geom_turn(aes(
    begin = line_begin,
    end = line_end,
    fill = load,
    colour = load),
    linewidth=0.4,
    alpha=0.8) +
  # geom_token(data = conv_token |> dplyr::filter(order=="first"),
  #            aes(x = line_relative_time,
  #                y = line_participant)) +
  geom_text(data = conv_token |> dplyr::filter(order=="only"),
            aes(x = line_relative_time,
                y=line_participant,
                label=token)) +
  scale_y_reverse(breaks = seq(1, max(conv_token$line_id))) +
  viridis::scale_colour_viridis(option="A") +
  viridis::scale_fill_viridis(option="A") +
  xlab("Time (ms)") +
  ylab("") +
  theme_turnPlot()
```

**Comment by mdingemanse** [comment]
small todos 

- [ ] don't forget to rename `init` to `format` in `workflows.Rmd`
- [ ] update `workflows.Rmd` to feature separate plots after #84 is implemented

**Comment by bvreede** [comment]
@mdingemanse this can be closed now that #89 is merged, right?

**Comment by mdingemanse** [comment]
Yes, thanks for noting!
